plugin/mdip.vim

```diff
@@ -60,8 +60,12 @@ function! SaveFileTMPMacOS(imgdir, tmpname) abort
 endfunction
 
 function! SaveFileTMP(imgdir, tmpname)
-    if has('mac')
-        return SaveFileTMPMacOS(a:imgdir, a:tmpname)
+    if has('unix')
+	"from mac https://superuser.com/a/193638/15231
+	let s:uname = system("uname")
```

Comment:
could you replace all tabs with spaces first?

Reply:
Done :)